verifier.py

```diff
@@ -160,7 +159,7 @@ def harvest_pages(self, json_list, url_end, page_class):
         :return: Null, but self.pages is populated.
         """
         for url in json_list:
-            if url.endswith("fail") == False and url.endswith(url_end):
+            if not url.endswith("fail"):
```

Comment:
No there's no reason. It's just that since we're leaving this implementation anyway I was only pushing the fix for the sake of records, so I didn't try that hard to fix anything else.